packages/core/src/components/DataTable/helpers/sortData.ts

```diff
@@ -25,10 +31,10 @@ export default function sortData<T extends GenericRow>(
 ): T[] {
   if (sortBy && keys.includes(sortBy)) {
     if (sortDirection === SortDirection.ASC) {
-      return list.slice().sort((a: T, b: T) => sortAsc(a.data[sortBy], b.data[sortBy]));
+      return list.slice().sort((a: T, b: T) => sort(a.data[sortBy], b.data[sortBy], true));
     }
 
-    return list.slice().sort((a: T, b: T) => sortDesc(a.data[sortBy], b.data[sortBy]));
+    return list.slice().sort((a: T, b: T) => sort(a.data[sortBy], b.data[sortBy]));
```

Comment:
Could keep `sortAsc()` and reverse? 

```ts
return list.slice().sort((a: T, b: T) => -sortAsc(a.data[sortBy], b.data[sortBy]));
```

Reply:
Ohh that's cool, I never knew you could use `-` to reverse.

But it doesn't work here cause then you'd end up with `null` back on top :(

Reply:
oh right. the nulls